anvio/docs/artifacts/kegg-metabolism.md

```diff
@@ -2,6 +2,9 @@ Output text files produced by %(anvi-estimate-metabolism)s that describe the pre
 
 Depending on the output options used when running %(anvi-estimate-metabolism)s, these files will have different formats.
 
+### How to get to this output
+![A beautiful workflow of metabolism reconstruction in anvi'o](../../images/metabolism_reconstruction.png)
```

Comment:
wow! you are so resourceful for digging this up! 🌟 

In truth, there is an arrow missing in this diagram, and I have put off fixing it for probably 2 months now 😅 my bad. I guess this is the push I needed, haha. Let me fix that and I will send you a new version of this workflow image!

Reply:
Okay, sounds good! 👍

Reply:
Ooooh, that's a good article right there! :)